src/modules/simulation/simulator_sih/sih.hpp

```diff
@@ -134,7 +134,7 @@ class Sih : public ModuleBase<Sih>, public ModuleParams
 	uORB::Subscription _actuator_out_sub{ORB_ID(actuator_outputs)};
 
 	// hard constants
-	static constexpr uint16_t NB_MOTORS = 6;
+	static constexpr uint16_t NB_MOTORS = 9;
```

Comment:
@mbjd Can we rename this to `NUM_ACTUATORS`?

Reply:
or even `NUM_ACTUATORS_MAX`

Reply:
[done](https://github.com/PX4/PX4-Autopilot/pull/24175/commits/eacbdc289f1bf00256701f226729577dc2b9cc3c)